crates/bevy_text/src/text.rs

```diff
@@ -1,10 +1,61 @@
 use bevy_asset::Handle;
 use bevy_math::Size;
 
-use crate::{Font, TextStyle};
+use crate::{Font, TextAlignment, TextStyle};
 
 #[derive(Debug, Default, Clone)]
 pub struct Text {
+    pub sections: Vec<TextSection>,
+    pub alignment: TextAlignment,
+}
+
+/// This is a transient helper type for basic text (text with only one section).
+/// Under the hood we require, use, construct and interact with the new "sectioned" [`Text`] type.
+/// Intended usage is:
+/// ```no_run
```

Comment:
the one you want in this case is `compile_fail` but I think it would be better to fix the example so that they compile...

Reply:
Ah, right. Any suggestions for how to make these compile?

Reply:
this pass doc tests for me: https://gist.github.com/mockersf/d9179ec1ef6df92753c473cb51fad416
and also adding to the dependencies of bevy_text:
```
[dev-dependencies]
bevy_ui = { path = "../bevy_ui", version = "0.4.0" }
```

Reply:
Ah of course. Thanks for this. Passes for me now. I've updated it.